appveyor.yml

```diff
@@ -3,21 +3,24 @@ environment:
   PATH: C:\%PYTHON%;C:\%PYTHON%\Scripts;%PATH%
   PYTHON: Python35
   matrix:
-    - TOX_ENV: lint
+    - TOX_ENV: py37
+    - TOX_ENV: py36
     - TOX_ENV: py35
-    - TOX_ENV: py34
-    - TOX_ENV: py33
     - TOX_ENV: py27
     - TOX_ENV: py
-      PYTHON: Python35-x64
+      PYTHON: Python37-x64
     - TOX_ENV: py
-      PYTHON: Python34-x64
+      PYTHON: Python36-x64
     - TOX_ENV: py
-      PYTHON: Python33-x64
+      PYTHON: Python35-x64
     - TOX_ENV: py
       PYTHON: Python27-x64
```

Comment:
Isn't AppVeyor duplicating environments? In example: `TOX_ENV: py37` is the same as `TOX_ENV: py`+`PYTHON: Python37-x64`.

Reply:
Great question. If you know the answer and the answer is that they are the same then I will remove. Otherwise let's take the original commit, https://github.com/sphinx-contrib/sphinxcontrib-versioning/commit/49addae676669959bcfe6840f6de6fd52652ffb7,  at face value  and assume that this is needed to test both 32-bit and 64-bit versions on Windows.

Reply:
It seems, from the AppVeyor logs, that they are duplicated environments. I would go ahead and remove them, but I am not really a maintainer. Ask @TimKam.

@TimKam I can help as a maintainer if you need a hand.

Reply:
Taking your word for it. Made the change.